content/articles/caa-record.markdown

```diff
@@ -35,8 +35,12 @@ See the [validations](#validation) section for more details about accepted value
 ## Policies restricting certificate issuance to specific types and CAs
 
 CAA records can control the issuance of single-name certificates, wildcard certificates, or both, though the `issue` and `issuewild` tags:
-- `issue` sets a policy for single-name certificate issuance.
-- `issuewild` sets a policy for wildcard and single-name certificate issuance and overrides any `issue` policy set on the same name.
+- `issue` sets a policy for for single-name **and** wildcard certificate issuance.
+- `issuewild` sets a policy for wildcard certificate issuance.
+
+<warning>
+The presence of `issuewild` overrides any CAA record with the `issue` tag in the same name for the same CA.
```

Comment:
```suggestion
The presence of `issuewild` overrides any CAA record with the `issue` tag for the same DNS qualified name.
```

That's actually irrelevant if it's the same CA or not. If an `issuewild` it is present, it takes precedence over any `issue` for wildcard.

```
example.com.  CAA 0 issue "ca1.example"
example.com.  CAA 0 issuewild "ca2.example"
```

The example above will allow ONLY ca1 to issue single-name, and ONLY ca2 to issue wildcard.

Reply:
What about:

> When requesting a certificate for a wildcard domain name, the presence of `issuewild` overrides any CAA record with the `issue` tag in the same name